pkg/model/mask.go

```diff
@@ -135,3 +157,17 @@ func UpdateService(src, dst *Service, mask Mask) error {
 
 	return nil
 }
+
+// UpdateTranslation updates the destination translation based on the source translation and field mask.
+func UpdateTranslation(src, dst *Translation, mask Mask) error {
+	// Prevent updating read-only fields like ID
+	if slices.ContainsFunc(mask, func(s string) bool {
+		return strings.EqualFold(s, "ID")
+	}) {
+		return errors.New("\"id\" is not allowed in field mask")
+	}
```

Comment:
Translation does not have any read-only fields, and also it does not have ID.
so possible values for mask is
```go
language
messages
original
```